00-HTML-CSS-basics/CSS/Ej 2.3 6 y 2.3 7

```diff
@@ -0,0 +1,7 @@
+6- I can add weight to the global font definition to win over the classes added by point 3 by creating a global selector
+
+*{
+    font-weight: 700;
+}
+
+7- Debo consultarlo. 
```

Comment:
que duda te quedo aca?

Reply:
7-Imagine there is a declaration like class=”oh-no-inline-styles” style=”background:red” and you need to change the background to green without changing the inline style. How could you accomplish this?
No supe como ganarle al inline-style , que según tengo entendido es el mas especifico .